src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala

```diff
@@ -3,17 +3,31 @@ package group.scala.karazin.circe.literal.extras
 import cats.implicits._
 import io.circe.syntax._
 import io.circe.parser
-import io.circe.{Json, JsonObject}
 import io.circe.{Json, JsonObject, Encoder, Codec}
 import org.scalacheck._
 import org.scalacheck.Prop._
+import group.scala.karazin.circe.literal.extras.arbitraries.instances.{given, _}
 
 class PlainPrimitiveEncodeSuite extends munit.ScalaCheckSuite:
 
   override def scalaCheckTestParameters =
     super.scalaCheckTestParameters
       .withMinSuccessfulTests(1)
 
+  property("raw json parsing") {
```

Comment:
Please follow the concept of the tests:
- should be two tests
- follow the structure
- follow the naming
- put the tests to the bottom of the file

Reply:
Done